src/App.tsx

```diff
@@ -1,20 +1,47 @@
-import { PeoplePage } from './components/PeoplePage';
 import { Navbar } from './components/Navbar';
 
 import './App.scss';
+import { Outlet } from 'react-router-dom';
+import { useContext, useEffect } from 'react';
+import { Person } from './types';
+import { getPeople } from './api';
+import { PeopleContext } from './contsxts/PeopleContext';
+import { ErrorContext } from './contsxts/ErrorContext';
+import { LoaderContext } from './contsxts/LoaderContext';
```

Comment:
The import path for `LoaderContext` is incorrect. It should be './contexts/LoaderContext' instead of './contsxts/LoaderContext'.